File: java/temperature-converter-program-ver3/Main.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		float tempC, tempF;
		Scanner scan = new Scanner(System.in);

		System.out.println("Enter a temp in C: ");
		tempC = scan.nextFloat();

		tempF = Calculate.celsiusToFahrenheit(tempC);

		System.out.println("Temp in F is: " + tempF);
	}
}

File: c++/temperature-converter-program-ver3/main.cpp
#include <iostream>
#include "functions.h"
using namespace std;

int main() {
	float tempC, tempF;
	
	cout<<"\nEnter temp C: ";
	cin>>tempC;
	
	tempF = celToFahren(tempC);
	
	cout<<"\nTemp in F: "<<tempF;
}